### PR TITLE
update icon: vyper (original, original-wordmark)

### PR DESCRIPTION
**Pull request by itsacoyote**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened

## Features
Updates Vyper logo to latest branding design. 
https://vyperlang.org/

## Notes
Removed `vyper-plain` and `vyper-plain-wordmark`. 
Updated `vyper-original` and `vyper-original-wordmark` and added aliases to cover `plain`.

### CODE REVIEW

File: icons/vyper/vyper-original-wordmark.svg
Comment (canaleal) [inline icons/vyper/vyper-original-wordmark.svg:1]: Great work on this icon! However, it needs to be optimized and formatted to be on one line with an extra empty line.

You can follow the steps in this [comment](https://github.com/devicons/devicon/issues/1279#issuecomment-1195433256) to optimize the SVG.

File: icons/vyper/vyper-original.svg
Comment (canaleal) [inline icons/vyper/vyper-original.svg:1]: This SVG also needs to be optimized, be on 1 line, and have an empty line.

**Review by canaleal — CHANGES_REQUESTED** [review]
Overall, great work!

All that's left is to optimize the SVGs and add "original-wordmark" to the SVG and Font entries inside the devicon.json file.

**Comment by itsacoyote** [comment]
> Overall, great work!
> 
> All that's left is to optimize the SVGs and add "original-wordmark" to the SVG and Font entries inside the devicon.json file.

Thanks for the suggestions! I tried to find as much of the documentation to make sure I was following things correctly, but had trouble finding the little bits. I'll make those changes 👍🏻

**Review by weh — APPROVED** [review]
Looking good, images are optimized. :+1:

**Review by Snailedlt — APPROVED** [review]
Looks good to me! ✔️ 
Thank you for the contribution! :rocket:

**Review by canaleal — APPROVED** [review]

**Pull request merged**